app/templates/index.js

```diff
@@ -0,0 +1,4 @@
+var h5bp = require('h5bp');
+
+var app = h5bp.createServer({ root: __dirname + '/public' });
```

Comment:
There isn't really a thing as a h5bp style guide so I think we should use the code style @sindresorhus prefers. Which is, if I'm correct, `h5bp.createServer({root: __dirname + '/public'});` (the spacing)